src/sentry/issues/ingest.py

```diff
@@ -189,3 +192,29 @@ def save_issue_from_occurrence(
         group_info = GroupInfo(group=group, is_new=is_new, is_regression=is_regression)
 
     return group_info
+
+
+def send_issue_occurrence_to_eventstream(
+    event: Event, occurrence: IssueOccurrence, group_info: GroupInfo
+) -> None:
+    event_group = event.for_group(group_info.group)
```

Comment:
Is `event_group` a `GroupEvent`? 😵 Might be less mind bendy if you call this `group_event` imo.

Reply:
I feel like my brain just randomly scrambled the order here 🤯